### DISCUSSION TITLE
I subscribed to the pro plan, even though I had a lot of database storage left

### DISCUSSION THREAD

**Discussion by mtalha101** [supabase/supabase#20560] [Questions] [answered]
I didn't know about the vacuum operations and my database crossed 500 MBs. I subscribed to the pro plan but later I figured out that I had a lot of space in my database after running the vacuum operation. My user base is pretty small and the pro plan is not needed for now. Is it possible to cancel the pro plan for now and get a refund?

**Comment by GaryAustin1** [comment, marked as answer]
This implies no from the pricing page FAQs.  
![image](https://github.com/supabase/supabase/assets/54564956/47978a93-2965-4510-b852-1e56700f335f)

I don't recall seeing anything else documented so unless someone from Supabase comes along you would have to ask support, or just downgrade and see.  There is no difference in performance between basic Pro and Free.

**Reply by mtalha101** [reply]
Thank you very much Sir!
